lwtime/epoch.go

```diff
@@ -16,9 +17,9 @@ func (epoch *Epoch) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-func (epoch *Epoch) MarshalJSON() ([]byte, error) {
-	// @afiune we might have problems changing the location :(
-	return epoch.ToTime().UTC().MarshalJSON()
+func (epoch Epoch) MarshalJSON() ([]byte, error) {
+	epochJson := fmt.Sprintf("%v", epoch.ToTime().UnixNano()/int64(time.Millisecond))
+	return []byte(epochJson), nil
```

Comment:
Do we have any tests to validate that this change won't affect other parts of the CLI? Can we add some?